v3/src/components/map/v2-map-importer.ts

```diff
@@ -52,12 +55,15 @@ export function v2MapImporter({v2Component, v2Document, insertTile}: V2TileImpor
     if (isPoints) {
       const {
         pointColor, strokeColor, pointSizeMultiplier,
+        grid, pointsShouldBeVisible, connectingLines
         /* Present in v2 layer model but not yet used in V3 layer model:
-        transparency, strokeTransparency, pointsShouldBeVisible,
-        grid, connectingLines,
+        transparency, strokeTransparency
         */
       } = v2LayerModel
-
+      // V2 point layers don't store their lat/long attributes, so we need to find them in the dataset
+      const {latId, longId} = latLongAttributesFromDataSet(data?.dataSet as IDataSet)
```

Comment:
Rather than casting here, I would add a test for `data?.dataSet` up above, i.e. something like:
```typescript
    // if we don't have a data set, then we can't add the layer
    if (!data?.dataSet) return
```
and then you shouldn't need the cast here or in line 100 below.

Reply:
Good idea!